app/src/renderer/components/staking/TabMyDelegations.vue

```diff
@@ -22,52 +28,85 @@
       <router-link :to="{ name: 'Validators' }">the validator list</router-link>
       to find other validators to delegate to.
     </div>
-    <div v-if="delegation.loaded && undelegatedValidators.length > 0">
-      <h3 class="tab-header">
-        Inactive Delegations
-        <i v-tooltip.top="unbondInfo" class="material-icons info-button"
-          >info_outline</i
+    <div v-if="delegation.loaded && unbondingTransactions.length > 0">
+      <h3 class="tab-header transactions">
+        Unbonding transactions
+        <i
+          v-tooltip.top="unbondTransactions"
+          class="material-icons info-button"
         >
+          info_outline
+        </i>
       </h3>
-      <table-validators :validators="undelegatedValidators" />
+      <div class="transactions">
+        <template v-for="transaction in unbondingTransactions">
+          <tm-li-stake-transaction
+            :transaction="transaction"
+            :validators="yourValidators"
+            :bonding-denom="bondingDenom"
+            :key="transaction.hash"
+            :url="validatorURL"
+          />
+        </template>
+      </div>
     </div>
   </div>
 </template>
 
 <script>
 import { mapGetters } from "vuex"
+import TmLiStakeTransaction from "../transactions/TmLiStakeTransaction"
 import TmDataMsg from "common/TmDataMsg"
 import TmDataLoading from "common/TmDataLoading"
 import TableValidators from "staking/TableValidators"
 import TmDataConnecting from "common/TmDataConnecting"
+import moment from "moment"
 
 export default {
   name: `tab-my-delegations`,
-  components: { TableValidators, TmDataMsg, TmDataConnecting, TmDataLoading },
+  components: {
+    TableValidators,
+    TmDataMsg,
+    TmDataConnecting,
+    TmDataLoading,
+    TmLiStakeTransaction
+  },
   data: () => ({
     bondInfo: `Validators you are currently bonded to`,
-    unbondInfo: `Your bonded validators in unbonding process`
+    unbondInfo: `Your bonded validators in unbonding process`,
+    unbondTransactions: `The transactions currently in unbonding period`,
+    validatorURL: `/staking/validators`
   }),
   computed: {
     ...mapGetters([
+      `allTransactions`,
       `delegates`,
       `delegation`,
       `committedDelegations`,
       `bondingDenom`,
       `connected`
     ]),
-    undelegatedValidators(
-      { delegates: { delegates }, delegation: { unbondingDelegations } } = this
-    ) {
-      return delegates.filter(
-        ({ operator_address }) => operator_address in unbondingDelegations
-      )
-    },
     yourValidators({ committedDelegations, delegates: { delegates } } = this) {
       return delegates.filter(
         ({ operator_address }) => operator_address in committedDelegations
       )
-    }
+    },
+    unbondingTransactions: ({ allTransactions, delegation } = this) =>
+      allTransactions
```

Comment:
will this show only active unbonding transactions or all unbonding transactions?

Reply:
Only unbonding, the filter underneath will remove all the others

Reply:
yes, but I have the feeling this will show unbonding transactions that are already passed the unbonding period. In this case they shouldn't show up anymore as the funds already have returned to the user. Am I mistaken?

Reply:
correct, I just added a helper and used it here

Reply:
NIce